test/util/testnode/genesis_options.go

```diff
@@ -34,7 +34,7 @@ func SetBlobParams(codec codec.Codec, params blobtypes.Params) GenesisOption {
 func ImmediateProposals(codec codec.Codec) GenesisOption {
 	return func(state map[string]json.RawMessage) map[string]json.RawMessage {
 		gs := v1.DefaultGenesisState()
-		gs.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1)))
+		gs.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(10)))
```

Comment:
As @evan-forbes alluded to, this line doesn't impact the `MinDeposit` used in production. The surrounding function `ImmediateProposals` is used in tests to make it easier + faster to pass governance proposals so it doesn't seem necessary to change this line.

```suggestion
		gs.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1)))
```

Reply:
Ok